numpy/core/src/multiarray/item_selection.c

```diff
@@ -2073,6 +2073,8 @@ PyArray_CountNonzero(PyArrayObject *self)
     char *data;
     npy_intp stride, count;
     npy_intp nonzero_count = 0;
+    int needs_api = 0;
```

Comment:
It appears to traditionally be `int` in numpy, according to grep. So I'll leave it.